frontend/src/common/PageWithTabs.tsx

```diff
@@ -19,30 +19,35 @@ export interface PageTab {
 
 export default function PageWithTabs({
   title,
+  subheading,
   tabs,
   actionButtonTitle = '',
   displayActionButton = false,
   actionButtonOnClick,
   requiredUrlParams = {},
   showCopyButton = false,
-  textToCopy = '',
+  titleToCopy = '',
+  subheadingToCopy = '',
   sourceModelId = '',
 }: {
   title: string
+  subheading?: string
   tabs: PageTab[]
   actionButtonTitle?: string
   displayActionButton?: boolean
   actionButtonOnClick?: () => void
   requiredUrlParams?: ParsedUrlQuery
   showCopyButton?: boolean
-  textToCopy?: string
+  titleToCopy?: string
+  subheadingToCopy?: string
```

Comment:
Is it necessary to have extra `...ToCopy` props since they're likely just repeats of existing props? I realise it's done this way already with the title?

Reply:
This is something I thought about, and you _are_ right when it comes to the title for models and data cards, but it's slightly different for the subheading. In my current implementation, the subheading is "ID: my-model-123", but I don't want to copy to clipboard the bit that says "ID:". Equally, we don't want to hard code "ID:" into the `PageWithTabs` component, because subheadings might not always be IDs, they can be any string value.